rxbinding-support-v4/src/main/java/com/jakewharton/rxbinding/support/v4/widget/NestedScrollViewScrollChangeEventOnSubscribe.java

```diff
@@ -0,0 +1,45 @@
+package com.jakewharton.rxbinding.support.v4.widget;
```

Comment:
ok, btw generateKotlin task does not work on Windows and I must wait for my colleague to run it and commit ;)